README.md

```diff
@@ -1178,6 +1178,7 @@ option      | default  | description
 id _or_ url |          | **Required.** Either the id or the url of the video.
 autopause   | `true`   | Pause this video automatically when another one plays.
 autoplay    | `false`  | Automatically start playback of the video. Note that this won’t work on some devices.
+background  | `false`  | Enable the player's background mode which hides the controls and autoplays the video.
```

Comment:
will `background=true` override the transparent property?

Reply:
in background mode the transparent mode is disabled yup.